src/display.ts

```diff
@@ -98,11 +98,30 @@ type FeatureInfo = {
 };
 
 const drawFeature = (svg: SVGSVGElement, face: Face, info: FeatureInfo) => {
-  const feature = face[info.name];
+  const feature = Object.assign({}, face[info.name]);
```

Comment:
Why this change? I think it doesn't do anything, but I may be missing something.

Reply:
The problem with that is you'd then have to generate every new face with a bunch of wrinkles and other old people features. Otherwise you'd have some that never age.

Ultimately you can get the same result either way, I just think it'd be more intuitive to have the base face object be the young version, and then add on stuff to represent aging. Because that's what actually happens as real humans age :)

Reply:
Another way of phrasing that is... you pick all the facial features you want for your guy. Some will appear/disappear based on age. If you say he has a bunch of wrinkles, he won't when he's young. If you say he has long hair, he might not when he's old. I think that makes enough sense for people to understand.